Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM maven:3-jdk-9-slim
-COPY . .
-EXPOSE 9090
-RUN mvn package -pl mauth-proxy -am -Dmaven.test.skip=true
-CMD ["sbt 'project proxy' run"]
```

Comment:
So the idea here is to replace this with an image, yes?  I was trying to convince @benton to put this all in artifactory, that way instead of building the project for docker, we could just pull a fat jar.  

That said, there still needs to be a Dockerfile somewhere.  Are you going to make another repo?

Reply:
publishing the fat jar is not recommended but we will publish the docker image, so anyone who wants to run it can use the docker image. I am discussing publishing of the docker image with @benton

Reply:
Okay, but I feel like somethings getting lost here.  I'll try to get a hold of you tomorrow.

1. As it stands now, the project doesn't build.  The point here is that I shouldn't have to install sbt in order to install mauth-proxy.  This is what I meant when I said we should change the base docker image.  Instead, you appear to have deleted docker.

2. If there is now an mauth-proxy docker image, that would be great but I'm wondering how you are building it because I can't get what you've got here to work.  It appears everything is broken for the moment.  I believe that happened during the last checkin